agency/test/hooks/schema.test.ts

```diff
@@ -0,0 +1,72 @@
+import { BadRequest } from '@feathersjs/errors';
+import { HookContext } from '@feathersjs/feathers';
+import { validateSchemaModel } from '../../src/utils/hooks/schemas';
+
+import schema from '../data/schema.json';
```

Comment:
`schema.json` does not seem to be committed

Reply:
Should be committed now